### ISSUE TITLE
404 Client Error

### ISSUE THREAD

**Issue by EdoARNAV** [smthemex/ComfyUI_EchoMimic#111]
Hi,

I get the attached error message when running the ComfyUI Workflow for the first time. Why?

**Echo_LoadModel**
404 Client Error. (Request ID: Root=1-67567d58-0d83b8430aff23d11edd801f;ca4f9684-e2bb-4624-85a1-23a643dd34a3)

Entry Not Found for url: https://huggingface.co/facebook/sapiens-pose-1b-torchscript/resolve/main/sapiens_1b_goliath_best_goliath_AP_640_torchscript.pt2.

![Screenshot 2024-12-09 073413](https://github.com/user-attachments/assets/f0de3361-83d7-4bd8-b5a2-9ee388534070)

Thanks,
Edo

**Comment by smthemex** [comment]
It's your net's error ,you can download from [here](https://huggingface.co/facebook/sapiens-pose-1b-torchscript/tree/main)
or [here ](https://hf-mirror.com/facebook/sapiens-pose-1b-torchscript/tree/main)

**Comment by EdoARNAV** [comment]
Wow, that was a fast response! :)
THANK YOU!
Where do I need to place it in my file system?
Edo

**Comment by smthemex** [comment]
![image](https://github.com/user-attachments/assets/345cdfc4-2e40-4af5-b517-d1205d371201)
check readme

**Comment by smthemex** [comment]
![image](https://github.com/user-attachments/assets/81029a09-ed16-405f-874e-7dd263e746ce)
Your file path is very strange, it's not the regular comfyUI model directory. This file is forced to point to models/echo_cimic
You can try modifying the code in the following two places to point to your model address
[54#](https://github.com/smthemex/ComfyUI_EchoMimic/blob/main/EchoMimic_node.py#L54) 
[L323](https://github.com/smthemex/ComfyUI_EchoMimic/blob/main/EchoMimic_node.py#L323)

**Comment by rouxianmantou** [comment]
why you use ` POSE_ESTIMATION_1B_T = "sapiens_1b_goliath_best_goliath_AP_640_torchscript.pt2"`  - `https://huggingface.co/facebook/sapiens-pose-1b-torchscript/blob/4caa2b2290255dc8963b5ead35fe3c6e761742aa/sapiens_1b_goliath_best_goliath_AP_640_torchscript.pt2` in your code?

`sapiens_1b_goliath_best_goliath_AP_639_torchscript.pt2` is currently in main branch..

**Comment by smthemex** [comment]
Yes, you pointed out the problem. It should be that I renamed the file name, and I will modify this code
